<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QWidget" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>细胞处理结果</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>19</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="15,2">
         <property name="spacing">
          <number>26</number>
         </property>
         <item>
          <widget class="ImageView" name="widget" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="check_btn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>9</height>
              </size>
             </property>
             <property name="text">
              <string>查看当前细胞处理过程</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="export_table_btn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>导出表格及图像</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
